Cargo.toml

```diff
@@ -54,7 +54,7 @@ minidom_ext = "1"
 minidom_writer = "1"
 num-traits = "0.2"
 pretty_assertions = "1"
-proj = { version = "0.24", optional = true } # libproj version used by 'proj' crate must be propagated to CI and makefile
+proj = { version = "0.25", optional = true } # libproj version used by 'proj' crate must be propagated to CI and makefile
```

Comment:
Does it involve a change of Proj C library version?

Reply:
It works with proj 8.1 so I suppose we don't need to change Proj C